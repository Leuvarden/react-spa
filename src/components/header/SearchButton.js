import React from 'react';

export default function SearchButton () {
    return <button className="search-panel__button-search">Search</button>;
}