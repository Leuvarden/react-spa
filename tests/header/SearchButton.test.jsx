import React from 'react';
import SearchButton from './../../src/components/header/SearchButton';


/* eslint-disable no-undef */
describe ('SearchButton', () => {

    it('renders as expected', () => {
        const button = shallow(<SearchButton />);
    
        expect(button).toMatchSnapshot();
    });

});