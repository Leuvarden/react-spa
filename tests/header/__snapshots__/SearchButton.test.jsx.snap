// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`SearchButton renders as expected 1`] = `
<button
  className="search-panel__button-search"
>
  Search
</button>
`;
